plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/SearchResultVersionXO.groovy

```diff
@@ -44,7 +45,7 @@ implements Comparable<SearchResultVersionXO>
     if (result == 0) {
       result = versionScheme.parseVersion(version ?: "0").compareTo(versionScheme.parseVersion(o.version ?: "0"))
       if (result == 0) {
-        result = path.compareTo(o.path)
+        result = path ?: "".compareTo(o.path ?: "")
```

Comment:
not sure this is logically equivalent for the first usage, if path is non-null you're not calling compareTo at all and setting result(int) to a String. Second usage achieves intention of avoiding null comparison